docs/repository-library-design.md

```diff
@@ -104,6 +116,28 @@ It would make sense for python-tuf to ship with at least one concrete
 Repository implementation: possibly a repo.py look alike. This implementation
 should not be part of the library but an example.
 
+### What a concrete Repository implementation must implement
+
+The current prototype API requires five methods that take care of access to
+metadata storage, expiry updates, version updates and signing.
+
+* **keyring()**: A property that returns the private key Mapping that should be
+  used for signing
+
+* **_load()**: Loads metadata from storage or cache. Is used by edit() and
+sign()
```

Comment:
You reader only learns about `sign()` in the next section in the context of your POC implementation. Maybe this is confusing here in the more generic part of the document? Any reason to not move this entire list down to the Details section?

Reply:
Hmm, this is all true, will change as suggested